plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityAge.java

```diff
@@ -119,46 +74,69 @@ public static void register() {
         // @tags
         // <EntityTag.age>
         // <EntityTag.is_baby>
-        // <EntityTag.is_age_locked>
-        // <EntityTag.ageable>
         // -->
         PropertyParser.registerMechanism(EntityAge.class, ListTag.class, "age", (prop, mechanism, param) -> {
             if (param.isEmpty()) {
                 mechanism.echoError("Missing value for 'age' mechanism!");
                 return;
             }
-            String input = param.get(0);
-            if (input.equalsIgnoreCase("baby")) {
-                prop.setAge(-24000);
-            }
-            else if (input.equalsIgnoreCase("adult")) {
-                prop.setAge(0);
-            }
-            else if (ArgumentHelper.matchesInteger(input)) {
-                prop.setAge(new ElementTag(input).asInt());
-            }
-            else {
-                mechanism.echoError("Invalid age '" + input + "': must be 'baby', 'adult', or a valid age number.");
+            String input = CoreUtilities.toLowerCase(param.get(0));
+            switch (input) {
+                case "baby" -> prop.setAge(-24000);
+                case "adult" -> prop.setAge(0);
+                default -> {
+                    if (!ArgumentHelper.matchesInteger(input)) {
+                        mechanism.echoError("Invalid age '" + input + "': must be 'baby', 'adult', or a valid age number.");
+                        return;
+                    }
+                    prop.setAge(new ElementTag(input).asInt());
+                }
             }
             if (param.size() > 1) {
-                input = param.get(1);
-                if (input.equalsIgnoreCase("locked")) {
-                    prop.setLock(true);
-                }
-                else if (input.equalsIgnoreCase("unlocked")) {
-                    prop.setLock(false);
+                BukkitImplDeprecations.oldAgeLockedControls.warn(mechanism.context);
+                if (!(prop.getEntity() instanceof Breedable breedable)) {
+                    return;
                 }
-                else {
-                    mechanism.echoError("Invalid lock state '" + input + "': must be 'locked' or 'unlocked'.");
+                switch (CoreUtilities.toLowerCase(param.get(1))) {
+                    case "locked" -> breedable.setAgeLock(true);
+                    case "unlocked" -> breedable.setAgeLock(false);
+                    default -> mechanism.echoError("Invalid lock state '" + param.get(1) + "': must be 'locked' or 'unlocked'.");
                 }
             }
         });
+
+        // <--[tag]
+        // @attribute <EntityTag.is_age_locked>
+        // @returns ElementTag(Boolean)
+        // @group properties
+        // @deprecated use 'age_locked'.
+        // @description
+        // Deprecated in favor of <@link tag EntityTag.age_locked>.
+        // -->
+        PropertyParser.registerTag(EntityAge.class, ElementTag.class, "is_age_locked", (attribute, prop) -> {
```

Comment:
Why not just add these as `deprecatedVariants` inputs to the new property?

Reply:
Correct me if I'm wrong, but these tags allowed `Ageable` entities and just returned `false` if they weren't `Breedable`, while the new property only allows entities that are `Breedable` in the first place; I.e. doing that would break using these tags on entities that are `Ageable` but not `Breedable`

Reply:
That's... probably fine to just change, considering that was never valid to do it just didn't directly error about it before.